scripts/net/run-sample-tests.sh

```diff
@@ -345,6 +342,26 @@ docker_exec ()
 
 	    wait_zephyr
 	    result=$?
+
+	    if [ $result -ne 0 ]
+	    then
+		break
+	    fi
+
+	    # TLS and SOCKS5, mosquitto TLS is already running
+	    start_docker "/usr/sbin/danted" || \
+		return $?
```

Comment:
can you please indent this (and similar start_zephyr long lines)? I won't hold this up, but it would help readability and be more consistent

Reply:
nvm it looks good in this preview, but it does look different with 'view changes'. Sorry for the confusion!

Reply:
Even the diff ouput in regular text sometimes shows the indentation to be off, when it is actually showing a tab. So yes, very inconsistent.